driver-core/src/test/java/com/datastax/driver/core/ConsistencyTest.java

```diff
@@ -15,716 +15,340 @@
  */
 package com.datastax.driver.core;
 
-import com.datastax.driver.core.exceptions.InvalidQueryException;
-import com.datastax.driver.core.exceptions.ReadTimeoutException;
-import com.datastax.driver.core.exceptions.UnavailableException;
-import com.datastax.driver.core.exceptions.WriteTimeoutException;
-import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;
-import com.datastax.driver.core.policies.DowngradingConsistencyRetryPolicy;
-import com.datastax.driver.core.policies.RoundRobinPolicy;
-import com.datastax.driver.core.policies.TokenAwarePolicy;
-import com.google.common.util.concurrent.Uninterruptibles;
+
+import org.scassandra.Scassandra;
+import org.scassandra.http.client.BatchExecution;
+import org.scassandra.http.client.PreparedStatementExecution;
+import org.scassandra.http.client.Query;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-import java.util.Arrays;
-import java.util.EnumSet;
 import java.util.List;
-import java.util.Set;
-import java.util.concurrent.TimeUnit;
-
-import static com.datastax.driver.core.CreateCCM.TestMode.PER_METHOD;
-import static com.datastax.driver.core.TestUtils.IP_PREFIX;
-import static com.datastax.driver.core.TestUtils.ipOfNode;
-import static java.util.Collections.emptyList;
-import static java.util.Collections.singletonList;
-import static org.testng.Assert.assertTrue;
-import static org.testng.Assert.fail;
-
-@SuppressWarnings("unused")
-@CreateCCM(PER_METHOD)
-@CCMConfig(
-        dirtiesContext = true,
-        createKeyspace = false,
-        config = {
-                // tests fail often with write or read timeouts
-                "phi_convict_threshold:5",
-                "read_request_timeout_in_ms:200000",
-                "write_request_timeout_in_ms:200000"
-        }
-)
-public class ConsistencyTest extends AbstractPoliciesTest {
+
+import static com.datastax.driver.core.TestUtils.nonQuietClusterCloseOptions;
+import static org.testng.Assert.*;
+
+public class ConsistencyTest {
+
 
     private static final Logger logger = LoggerFactory.getLogger(ConsistencyTest.class);
+    private ScassandraCluster sCluster;
 
-    private Cluster.Builder tokenAwareRoundRobin() {
-        return Cluster.builder()
-                .withSocketOptions(new SocketOptions().setReadTimeoutMillis(300000))
-                .withLoadBalancingPolicy(new TokenAwarePolicy(new RoundRobinPolicy()));
-    }
+    @BeforeClass
+    public void setUp() {
```

Comment:
would also be good to stop the Scassandra cluster after the test exits, it looks like now it still runs